src/domain/api/__tests__/client.test.ts

```diff
@@ -21,11 +21,13 @@ describe('graphql client', () => {
 
     try {
       await client.query({
-        query: childrenQuery,
+        query: profileQuery,
```

Comment:
Removing childrenQuery since it was only used in this test. Always nice to clean up the attic a bit.